src/action/common/configure_shell_profile.rs

```diff
@@ -8,19 +8,29 @@ use tokio::task::JoinSet;
 use tracing::{span, Instrument, Span};
 
 // Fish has different syntax than zsh/bash, treat it separate
-const PROFILE_FISH_SUFFIX: &str = "conf.d/nix.fish";
+const PROFILE_FISH_CONFD_SUFFIX: &str = "vendor_conf.d/nix.fish";
+/**
+ Each of these are common values of $__fish_vendor_confdir,
+under which Fish will look for a file named
+[`PROFILE_FISH_CONFD_SUFFIX`].
+
+More info: https://fishshell.com/docs/3.3/index.html#configuration-files
+*/
+const PROFILE_FISH_VENDOR_CONFD_PREFIXES: &[&str] = &["/usr/share/fish/", "/usr/local/share/fish/"];
 
+const PROFILE_FISH_VENDOR_CONFD_SUFFIX: &str = "conf.d/nix.fish";
```

Comment:
I think you swapped these names; the name reference vendor_conf.d, but this one is just conf.d.

Reply:
Oh my goodnesss